org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/binding/wizard/project/SonarProjectSelectionWizardPage.java

```diff
@@ -41,7 +41,7 @@ public class SonarProjectSelectionWizardPage extends AbstractProjectBindingWizar
   private Binding projectTextBinding;
 
   public SonarProjectSelectionWizardPage(ProjectBindingModel model) {
-    super("remote_project_page", "Choose the SonarQube/SonarCloud project", model, 1);
+    super("sonar_project_page", "Choose the SonarQube/SonarCloud project", model, 1);
```

Comment:
Does changing to `sonar_project_page` have any impact? Like reference in the `plugin.xml`?

Reply:
AFAIR, this key is used to automatically persist some settings about the wizard page (like default position of the dialog). I don't think it is a breaking change.